recipes/install_ui.rb

```diff
@@ -27,6 +28,22 @@
 
 include_recipe "abiquo::certificate"
 
+case node['abiquo']['profile']
+when 'monolithic'
+    node.set['abiquo']['ui_proxies'] = node['abiquo']['ui_proxies'].merge({
+        '/api' => "ajp://localhost:#{node['abiquo']['tomcat']['ajp-port']}/api",
+        '/legal' => "ajp://localhost:#{node['abiquo']['tomcat']['ajp-port']}/legal",
+        '/am' => "ajp://localhost:#{node['abiquo']['tomcat']['ajp-port']}/am",
+        '/m' => "http://localhost:#{node['abiquo']['tomcat']['http-port']}/m"
+    })
+when 'server'
+    node.set['abiquo']['ui_proxies'] = node['abiquo']['ui_proxies'].merge({
+        '/api' => "ajp://localhost:#{node['abiquo']['tomcat']['ajp-port']}/api",
+        '/legal' => "ajp://localhost:#{node['abiquo']['tomcat']['ajp-port']}/legal",
+        '/m' => "http://localhost:#{node['abiquo']['tomcat']['http-port']}/m"
+    })
```

Comment:
The merges should be inverted to make sure the values configured by the user take precedence in case of key collision.